opendbc/car/hyundai/interface.py

```diff
@@ -33,6 +33,10 @@ def _get_params(ret: structs.CarParams, candidate, fingerprint, car_fw, alpha_lo
     cam_can = CanBus(None, fingerprint).CAM
     lka_steering = 0x50 in fingerprint[cam_can] or 0x110 in fingerprint[cam_can]
     CAN = CanBus(None, fingerprint, lka_steering)
+    print(cam_can, lka_steering, CAN)
+
+    # assert 0x50 not in fingerprint[CAN.ACAN] and 0x110 not in fingerprint[CAN.ACAN]
+    # assert 0x50 not in fingerprint[CAN.ECAN] and 0x110 not in fingerprint[CAN.ECAN]
```

Comment:
can't do this, some cars have garbage LKAS_ALT on ECAN (not real LKAS_ALT I'm pretty sure)